app/views/prior_authority/steps/submission_confirmation/show.html.erb

```diff
@@ -15,7 +15,7 @@
     <p class="govuk-body"><%= t('.confirmation_email_sent')%></p>
       <h2 class="govuk-heading-m"><%= t('.next_steps.title')%></h2>
       <p class="govuk-body"><%= t('.next_steps.process_1')%></p>
-      <p class="govuk-body"><%= t('.next_steps.process_2')%></p>
+      <p class="govuk-body"><%= t('.next_steps.process_2_html')%></p>
```

Comment:
so you now have `<p class="govuk-body">` nested inside `<p class="govuk-body">` as it is in the translation as well... :)

Reply:
Good shout. missed it completely. Fixing now